### PR TITLE
Change warning on EVM local transfer

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Change warning on EVM local transfer

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

스크린샷 2023-05-11 오전 10 05 09

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Thank you! Can the checkbox stay next to the consent message?

**Comment by sirius651** [comment]
> Thank you!
> 
> * Can the checkbox stay next to the consent message?
> * Also mentioned on Slack and sorry, this only applies for ASTR local transfer, when EVM address is input. (ledger doesn't support EVM withdraw)
스크린샷 2023-05-11 오후 3 24 07

@Kahonnohak Fixed like this & only visible on EVM address input

**Comment by Kahonnohak** [comment]
Oh no, I messed up I'm very sorry...
EVM deposit is not supported, which means, this needs to be shown
- When connected to EVM
- And local transfer ASTR to Native account.

1) So now the Native side needs to go back to what it was before.
2)  And apply the change above to EVM side

Sorry for this. at the moment EVM side has issue since I think you add this change so please make sure EVM and Native have those warnings. and not go on the other warnings.

Thank you and sorry x100! 
Screenshot 2023-05-11 at 07 44 26

**Comment by impelcrypto** [comment]
This has to have fixed
image

**Comment by impelcrypto** [comment]
From native account too
image

**Comment by sirius651** [comment]
> Thank you and sorry x100!

스크린샷 2023-05-11 오후 4 38 03

@Kahonnohak does that mean show warning like this as it was?

**Comment by sirius651** [comment]
스크린샷 2023-05-11 오후 4 58 57

so fixed as only visible this warning message if EVM - toAddress is ss58. 
just let me know if it's wrong @Kahonnohak

**Review by Kahonnohak — APPROVED** [review]
Thank you so much and sorry for miscommunications!

File: src/components/assets/transfer/LocalTransfer.vue [hunk 1]
```diff
@@ -132,10 +132,23 @@
...
                 {{ $t('assets.modals.notSendToExchanges') }}
               </span>
             </div>
+            <span
+              v-if="isH160 && !isNativeToEvm"
+              :class="isChecked ? 'color--gray1' : 'color--not-checked'"
+            >
+              <span class="text--evm-warning">{{ $t('assets.modals.understandWarning') }}</span>
+              <div class="text--warn">
+                <span class="text--dot">・</span> {{ $t('assets.modals.notDestIsLedgerAccount') }}
```
Comment (impelcrypto) [inline src/components/assets/transfer/LocalTransfer.vue:145]: @Kahonnohak Maybe just change to "The destination address is neither a ledger native account nor an exchange address"?

Comment (Kahonnohak) [reply src/components/assets/transfer/LocalTransfer.vue:145]: No objection! But that means we could use a "Warning" EVM style.

"The destination address is neither a ledger native account nor an exchange address. I understand that if I do so, the funds will likely be lost."

Screenshot 2023-05-11 at 09 55 42

**Review by bobo-k2 — APPROVED** [review]

File: src/components/assets/transfer/LocalTransfer.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/assets/transfer/LocalTransfer.vue:145]: > No objection! But that means we could use a "Warning" EVM style.

Yes, I think this is better than original design

Comment (Kahonnohak) [reply src/components/assets/transfer/LocalTransfer.vue:145]: @sirius651 Revise again ｍ(｡≧Д≦｡)ｍ

**Review by niklabh — APPROVED** [review]

File: src/components/assets/transfer/LocalTransfer.vue (hunk 1 above)
Comment (sirius651) [reply src/components/assets/transfer/LocalTransfer.vue]: 스크린샷 2023-05-11 오후 7 42 17
Fixed like this

**Comment by Kahonnohak** [comment]
Thank you Will! Good to go now!!

**Pull request merged**